### DISCUSSION TITLE
v0.18.1

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1279] [Releases]
## What's Changed
* [Bug] Use server list when running manual speedtests by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1276
* [Bug] Don't proceed with notifications if absolute threshold disabled by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1277
* Release v0.18.1 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1278

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.18.0...v0.18.1

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.18.1'>v0.18.1</a>.</em>

**Comment by seh2000** [comment]
Installed and Run Speedtest is now using same server. :)
